src/views/AffiliatesBanner.tsx

```diff
@@ -22,10 +24,55 @@ import { openDialog } from '@/state/dialogs';
 export const AffiliatesBanner = () => {
   const dispatch = useAppDispatch();
   const stringGetter = useStringGetter();
+  const { isTablet } = useBreakpoints();
   const { affiliateProgram } = useURLConfigs();
 
   const background = useAppSelector(getGridBackground);
 
+  const titleString = stringGetter({
+    key: STRING_KEYS.EARN_FOR_EACH_TRADER,
+    params: { AMOUNT_USD: AFFILIATES_EARN_PER_MONTH.toLocaleString() },
+  });
+  const descriptionString1 = stringGetter({
```

Comment:
[suggestion] we can probably find a more descriptive name for these variables

Reply:
or if the numbering is intended to flag that the ordering here is always important, maybe we can combine them?
```
const descriptionString = (
  <>
    {descriptionString1} <br />
    {descriptionString2}
  </>
);
```